src/inference/src/cpp_interfaces/interface/ie_iplugin_internal.cpp

```diff
@@ -340,75 +342,96 @@ std::unordered_set<std::string> GetSupportedNodes(
     }
 
     auto transformed_model = model->clone();
+
+    // Cleanup fused names if there are present in original model
```

Comment:
Why do we need it?

Reply:
> Why do we need it?

For details "Rules changed: fused_names pull back" mail thread from @slyalin